--- navit/route.c	2022-02-16 16:55:55.694015310 +0100
+++ navit/route.c	2022-02-16 16:55:55.962149664 +0100
@@ -4263,7 +4263,7 @@
     dbg(lvl_debug,"enter");
     switch (attr->type) {
     case attr_callback:
-        callback_list_remove(this_->cbl2, attr->u.callback);
+//        callback_list_remove(this_->cbl2, attr->u.callback);
         return 1;
     case attr_vehicle:
         this_->v=NULL;
