--- navit/textures/CMakeLists.txt
+++ navit/textures/CMakeLists.txt
@@ -30,7 +30,7 @@ macro(convert_to_png TEXTURE_INPUT TEXTURE_OUTPUT SCALE)
 		else()
 			set(SCALE_ARGS)
 		endif()
-		set(COMMAND_ARGS --export-type=png --export-background-opacity=0 ${SCALE_ARGS} --export-filename=${TEXTURE_OUTPUT} ${TEXTURE_INPUT})
+		set(COMMAND_ARGS --export-background-opacity=0 ${SCALE_ARGS} --export-png=${TEXTURE_OUTPUT} ${TEXTURE_INPUT})
 	else()
 		message(FATAL_ERROR "no svg2png converter defined here.")
 	endif()
