--- navit/icons/CMakeLists.txt
+++ navit/icons/CMakeLists.txt
@@ -30,7 +30,7 @@ macro(convert_to_png IMAGE_INPUT IMAGE_OUTPUT SCALE)
 		else()
 			set(SCALE_ARGS)
 		endif()
-		set(COMMAND_ARGS --export-type=png --export-background-opacity=0 ${SCALE_ARGS} --export-filename=${IMAGE_OUTPUT} ${IMAGE_INPUT})
+		set(COMMAND_ARGS --export-background-opacity=0 ${SCALE_ARGS} --export-png=${IMAGE_OUTPUT} ${IMAGE_INPUT})
 	else()
 		message(FATAL_ERROR "no svg2png converter defined here.")
 	endif()
 
